css/main.css

```diff
@@ -0,0 +1,59 @@
+body{
+    margin : 0;
+}
+.navigation ul li{
+    float: left;
+    list-style: none;
+    color: white;
+    padding:10px;
+    padding-top: 3px;
+   
+    
+    
+}
+.base{
+    text-decoration: underline;
+}
+.navigation{
+    background-color: #5e5e5e;
+    overflow:auto;
```

Comment:
why an `overflow: auto;` attribute

Reply:
i used overflow,since after using float ,Background color was not visible in the navigation bar